content/_data/changelogs/lts.yml

```diff
@@ -6978,18 +6946,6 @@
   date: 2022-04-06
   changes:
 
-  - type: rfe
-    category: rfe
-    authors:
-      - slide
-    issue: 68170
-    pr_title: "[JENKINS-68170] Show warning dialog if Java 8 is selected"
-    references:
-      - issue: 68170
-      - url: https://github.com/jenkinsci/packaging/pull/306
-        title: pull 306
-    message: |-
-      Show warning dialog if Java 8 is selected in Windows installer.
```

Comment:
Why are you removing entries from past releases?

Reply:
Hi @daniel-beck, sorry about that! I was working with @MarkEWaite on removing the minor items, and found the entries in multiple places thinking they were duplicates. They were removed in an effort to clean things up, but can see why that was a mistake.